data_steward/curation_logging/slack_logging_handler.py

```diff
@@ -0,0 +1,44 @@
+# Python imports
+import os
+import logging
+
+# Project imports
+from utils.slack_alerts import post_message, check_channel_and_token, SlackConfigurationError, SLACK_CHANNEL, SLACK_TOKEN
+
+
+class SlackLoggingHandler(logging.Handler):
+    """
+     Logging handler to send messages to a Slack Channel.
+    """
+
+    def __init__(self):
+        super().__init__(level=logging.WARNING)
+
+    def emit(self, record):
+
+        try:
+            if not _is_raised_from_itself(record):
+                post_message(record.msg % record.args)
+        except SlackConfigurationError:
+            logging.exception(
+                'Slack is not configured for posting messages, refer to playbook.'
+            )
+            raise
+
+def _is_raised_from_itself(record):
+    return record.module in __name__
+
+
+def initialize_slack_logging():
+    """
+    Setup Slack logging
+    """
+    if SLACK_TOKEN in os.environ and SLACK_CHANNEL in os.environ:
+        if check_channel_and_token:
```

Comment:
I think we can just check the channel directly since it internally checks whether the token is authorized. 

```suggestion
        if is_channel_available():
```